src/modules/components/proposal/proposalActions/proposalActionsActionVerfication/proposalActionsActionVerification.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Given that this is an internal component that is not exported, we don't need the stories for it